boards/arm/adafruit_kb2040/adafruit_kb2040-pinctrl.dtsi

```diff
@@ -47,4 +47,10 @@
 			input-enable;
 		};
 	};
+
```

Comment:
And please add a description as well to commit adding the driver. It would be useful to describe a bit the PIO interface and how it can be used to output a signal compatible with WS2812 LED strip controllers.

Reply:
I added information to commit message about setting that used to verify this patch.